### PR TITLE
Relicense `MIT OR Apache-2.0`

### PR DESCRIPTION
**Pull request by jackpot51**
Relicense from MPL-2.0 to dual license MIT or Apache 2.0. This license is chosen to more easily allow the flow of code from this project from and to other Rust projects.

- [x] @jackpot51
- [x] @kettle11 
- [x] @WatchMkr

### CODE REVIEW

**Comment by jackpot51** [comment]
I license past and future contributions under the dual MIT OR Apache-2.0, allowing licensees to chose any one at their option.

**Comment by ids1024** [comment]
The `license` field in `Cargo.toml` should also be updated.

**Comment by jackpot51** [comment]
> The `license` field in `Cargo.toml` should also be updated.

Done, and squashed.

**Comment by kettle11** [comment]
I license past and future contributions under the dual MIT OR Apache-2.0, allowing licensees to chose any one at their option.

Thanks for making this change!

File: Cargo.toml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name = "cosmic-text"
 version = "0.1.0"
 authors = ["Jeremy Soller <jeremy@system76.com>"]
 edition = "2021"
-license = "MPL2"
+license = "MIT / Apache-2.0"
```
Comment (crawfxrd) [inline Cargo.toml:6]: Replace `/` with `OR` to make it a valid SPDX expression.

**Review by WatchMkr — APPROVED** [review]

**Pull request merged**